README.md

```diff
@@ -212,3 +212,26 @@ const proc = x('ls');
 
 proc.exitCode; // number (e.g. 1)
 ```
+
+## Comparison with other libraries
+
+`tinyexec` aims to provide a lightweight layer on top of Node's own
+`child_process` API.
+
+Some clear benefits are that `tinyexec` will be much lighter, have a much
```

Comment:
It could be clearer here that you're talking about other libraries and not child_process
```suggestion
Some clear benefits compared to other libraries are that `tinyexec` will be much lighter, have a much
```